### PR TITLE
deleted default readme from create-react-app

### PR DESCRIPTION
**Pull request by jimcwx**
HELP

### CODE REVIEW

**Comment by jimcwx** [comment]
HI
-JESSE

File: README.md [hunk 1]
```diff
@@ -1,68 +1,24 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
 
-In the project directory, you can run:
 
-### `npm start`
+//*******************************************************//
```
Comment (mleandres) [inline README.md:5]: this is okay not existing OR you can use some Markdown to style things
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

you can use a horizontal rule here like this:

---

and also headers like this
## this is a header
### this is a less important header

**_formatting_**

File: README.md [hunk 2]
```diff
@@ -1,68 +1,24 @@
...
 
-### `npm test`
+Select up to two champion types
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+After the champion portraits are displayed you can click on the portrait to enter a detailed champion descriptions page
 
-### `npm run build`
+~~~~~~~~~~~WARNING~~~~~~~~~~~~
```
Comment (mleandres) [inline README.md:15]: this could be a header

File: README.md [hunk 3]
```diff
@@ -1,68 +1,24 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (mleandres) [inline README.md:1]: can this line be at the bottom?

**Pull request merged**